tools/certa/src/utils/CoverageUtils.ts

```diff
@@ -59,7 +59,7 @@ export function writeCoverageData(coverageData: any): void {
   if (!fs.existsSync(nycTempDirAbsolute))
     throw new Error(`Cannot save coverage data - nyc temp directory "${nycTempDirAbsolute}" does not exist.`);
 
-  // Use uuid/v4 to generate a unique filename, just like `nyc` does.
-  const coverageFileName = path.join(nycTempDirAbsolute, `${uuid.v4()}.json`);
+  // Use Guid to generate a unique filename, just like `nyc` does.
+  const coverageFileName = path.join(nycTempDirAbsolute, `${Guid.createValue()}.json`);
```

Comment:
this is a 3 line function, you can just copy it here if that's easier/less gross.

Reply:
I did debate this while removing the dep locally. Only reason I opted not to was just in case if we forget to update this file later on when when we switch to crypto.randomUUID. Thought it'd be easier to just search for all uses of Guid.createValue and update everything together then. 

I will update and just make sure to note in the issue to update this file too.

Reply:
all that's needed here is to add some randomess to the filename - it doesn't need a UUID at all, if that helps.

Reply:
great point, thx. updated